src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/GCHandle.CoreRT.cs

```diff
@@ -9,7 +9,7 @@ public partial struct GCHandle
 
         private static void InternalFree(IntPtr handle) => RuntimeImports.RhHandleFree(handle);
 
-        private static object InternalGet(IntPtr handle) => RuntimeImports.RhHandleGet(handle);
+        internal static object InternalGet(IntPtr handle) => RuntimeImports.RhHandleGet(handle);
```

Comment:
Instead of making this `internal`, I think it would be better to either call the public method on GCHandle, or call `RuntimeImports.RhHandleGet` directly.